pkg/pillar/cmd/volumemgr/initialvolumestatus.go

```diff
@@ -27,62 +24,22 @@ import (
 // Really a constant
 var nilUUID = uuid.UUID{}
 
-// appRwVolumeName - Returns name of the image ( including parent dir )
-// Note that we still use the sha in the filename to not impact running images. Otherwise
-// we could switch this to imageID
-// XXX other types of volumes might want a different name.
-func appRwVolumeName(sha256, uuidStr string, purgeCounter uint32, format zconfig.Format,
-	origin types.OriginType, isContainer bool) string {
-
-	purgeString := ""
-	if purgeCounter != 0 {
-		purgeString = fmt.Sprintf("#%d", purgeCounter)
-	}
-	if isContainer {
-		return fmt.Sprintf("%s-%s%s", sha256, uuidStr, purgeString)
-	}
-	if origin != types.OriginTypeDownload {
-		log.Fatalf("XXX unsupported origin %v", origin)
-	}
-	formatStr := strings.ToLower(format.String())
-	return fmt.Sprintf("%s/%s-%s%s.%s", rwImgDirname, sha256,
-		uuidStr, purgeString, formatStr)
-}
-
-// parseAppRwVolumeName - Returns rwImgDirname, sha256, uuidStr, purgeCounter
-func parseAppRwVolumeName(image string, isContainer bool) (string, string, string, uint32) {
+// parseAppRwVolumeName - Returns rwImgDirname, volume uuid, generationCounter
+func parseAppRwVolumeName(image string) (string, string, uint32) {
 	// VolumeSha is provided by the controller - it can be uppercase
 	// or lowercase.
```

Comment:
Comment is off, but the RE is fine AFAICT

Reply:
Removed.